# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt
# Commit: 00d82acfaa2f
# Full Hash: 00d82acfaa2f56095e5033a6905427d43b95eccb
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 4ba88410cdf6 -r 00d82acfaa2f third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt
--- a/third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt	Fri Jan 27 18:35:14 2023 +0000
+++ b/third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt	Fri Jan 27 18:35:14 2023 +0000
@@ -1,9 +1,9 @@
-file 'star.png'
-duration 0.1
-file 'star90.png'
-duration 0.1
-file 'star180.png'
-duration 0.1
-file 'star270.png'
-duration 0.1
-file 'star.png'
+file 'star.png'
+duration 0.1
+file 'star90.png'
+duration 0.1
+file 'star180.png'
+duration 0.1
+file 'star270.png'
+duration 0.1
+file 'star.png'